cypress/integration/login.js

```diff
@@ -8,11 +8,32 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Valentinas Kasteckis"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="Team Lead"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Valentinas Kasteckis');

```

Comment:
You can delete this line and one bellow, because you already check for user name and tabs count in other test (T-2). This minor but still, its always better to not check same thing twice :)